### PR TITLE
Occitan version update

### PR DESCRIPTION
**Pull request by Quent-in**
| Q             | A
| ------------- | ---
| Bug fix?      | yes|no
| New feature?  | yes|no
| BC breaks?    | yes|no
| Deprecations? | yes|no
| Tests pass?   | yes|no
| Documentation | yes|no
| Translation   | yes|no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

If you need to write dates in full letter you might be interested in https://github.com/fightbulc/moment.php
It comes with lots of languages ;)
Q: where are the "previous" and "next" button text? Can't find them to translate them.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ footer:
         social: 'Social'
         powered_by: 'propulsat per'
         about: 'A prepaus'
-    # stats: Since %user_creation% you read %nb_archives% articles. That is about %per_day% a day!
+        stats: "Dempuèi %user_creation% avètz legit %nb_archives% articles. Es a l'entorn de %per_day% per jorn !"
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: Wrong, it should have only 4 spaces, not 8

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 2]
```diff
@@ -298,14 +298,13 @@ quickstart:
         readability: 'Migrar dempuèi Readability'
         instapaper: 'Migrar dempuèi Instapaper'
     developer:
-        title: 'Pels desvolopadors'
-        # description: 'We also thought to the developers: Docker, API, translations, etc.'
+        title: 'Pels desvolopaires'
+        description: 'Avèm tanben pensat als desvolopaires : Docker, API, traduccions, etc.'
         create_application: 'Crear vòstra aplicacion tèrça'
-        # use_docker: 'Use Docker to install wallabag'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: You should keep that line

**Comment by j0k3r** [comment]
What previous & next are you talking about?

**Comment by Quent-in** [comment]
in the home page, for the pagination, I have them in English

**Comment by j0k3r** [comment]
Oh that, it's not wallabag, it's an other package: https://github.com/whiteoctober/WhiteOctoberPagerfantaBundle/tree/master/Resources/translations

**Comment by Quent-in** [comment]
I've just made a PR for OC version there, thanks!

**Comment by j0k3r** [comment]
@Quent-in could you please fix the two comments I reported? Thanks

**Comment by nicosomb** [comment]
Maybe we can merge this PR into master?

**Comment by tcitworld** [comment]
Does it matter ?

**Pull request merged**